src/resolvers/Mutation/updateAdvertisement.ts

```diff
@@ -82,12 +83,28 @@ export const updateAdvertisement: MutationResolvers["updateAdvertisement"] =
       );
     }
 
```

Comment:
Here superadmins, and admins of any organization can update any advertisement, is this expected?